spec/rubocop/cop/style/if_unless_modifier_of_if_unless_spec.rb

```diff
@@ -0,0 +1,90 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::IfUnlessModifierOfIfUnless do
+  include StatementModifierHelper
+
+  subject(:cop) { described_class.new(config) }
+  let(:config) do
+    RuboCop::Config.new({})
+  end
+
+  it 'provides a good error message' do
+    source = 'condition ? then_part : else_part unless external_condition'
+    inspect_source(cop, source)
+    message = cop.offenses.first.message
```

Comment:
You can directly check `cop.messages`. See other specs. It's best to check the entire message.